packages/client/internals/SlideWrapper.ts

```diff
@@ -1,26 +1,14 @@
-import { useVModel } from '@vueuse/core'
-import { computed, defineComponent, h, provide, ref, toRef } from 'vue'
-import type { RenderContext } from '@slidev/types'
-import { injectionActive, injectionClicks, injectionClicksDisabled, injectionClicksElements, injectionCurrentPage, injectionOrderMap, injectionRenderContext, injectionRoute } from '../constants'
+import { defineComponent, h, provide, ref, toRef } from 'vue'
+import type { PropType } from 'vue'
+import type { ClicksContext, RenderContext } from '@slidev/types'
+import { injectionActive, injectionClicks, injectionCurrentPage, injectionRenderContext, injectionRoute } from '../constants'
 
 export default defineComponent({
   name: 'SlideWrapper',
   props: {
     clicks: {
-      type: [Number, String],
-      default: 0,
-    },
-    clicksElements: {
-      type: Array,
-      default: () => [] as Element[],
-    },
-    clicksOrderMap: {
-      type: Map,
-      default: () => new Map<number, HTMLElement[]>(),
-    },
-    clicksDisabled: {
-      type: Boolean,
-      default: false,
+      type: Object as PropType<ClicksContext>,
```

Comment:
Maybe let's name this prop `clicksContext` to be explicit and avoid confusing with the clicks `$clicks` which contains a number.